pkg/constants/constants.go

```diff
@@ -40,8 +40,10 @@ const (
 	EksaLicenseName        = "eksa-license"
 	EksaPackagesName       = "eksa-packages"
 
-	DefaultRegistry            = "public.ecr.aws"
-	CloudstackAnnotationSuffix = "cloudstack.anywhere.eks.amazonaws.com/v1alpha1"
+	DefaultRegistryMirrorKey    = "base"
+	DefaultRegistry             = "public.ecr.aws"
+	DefaultPackageRegistryRegex = "783794618700.dkr.ecr.*.amazonaws.com"
+	CloudstackAnnotationSuffix  = "cloudstack.anywhere.eks.amazonaws.com/v1alpha1"
```

Comment:
Whats the rationale for housing these constants here? The `constants` package is turning into a dumping ground for totally unrelated things beyond the fact their definitions are typed with `const`.